ote/src/clj/ote/services/transport.clj

```diff
@@ -271,6 +271,13 @@
 
     (external/save-companies db new-data)))
 
+(defn- handle-companies
```

Comment:
I think this should be called something else to better reflect what it does (clears company CSV URL depending on source type). The name is too generic and parameter is called `data`

Reply:
Yes, you are right. Renamed